operator/cluster/clusterlogic.go

```diff
@@ -40,7 +40,7 @@ import (
 )
 
 // AddCluster ...
-func AddCluster(clientset *kubernetes.Clientset, client *rest.RESTClient, cl *crv1.Pgcluster, namespace string, primaryPVCName string) error {
+func AddCluster(clientset *kubernetes.Clientset, client *rest.RESTClient, cl *crv1.Pgcluster, namespace, primaryName string) error {
```

Comment:
I strongly disagree with the change to this function definition. The `primaryName` should already be set in `cluster.Spec.Name` and we should continue to pass in the `pvcName`

Reply:
Understood. I believe now with the updated annotation keeping track of the current primary, I should be able to revert this back to the previous function definition and grab that value from the annotation. 

One note, `cluster.Spec.Name` will not necessarily match the PVC name for clusters that have been through a failover or restore event (technically, you could failover to a replica, then fail BACK over to the original pod as the primary, thus realigning the PVC and cluster name, so not all 'failed-over' clusters have this issue). That's why some sort of current primary value is required in this case.

Reply:
Yes -- but that's why we're keeping that in the annotation :smile:

Reply:
Yep, :+1: